demos/app.js

```diff
@@ -49,10 +49,12 @@ app.engine('.html', new PageKitHandlebars({
 app.use('/public', express.static(path.join(__dirname, '../public'), { redirect: false }));
 
 app.get('/', (req, res) => {
+	const state = req.query.state || 'default';
+	const cardState = fixtures[state];
```

Comment:
**Suggestion:** Sorry to be a pain but I think this might be over complicating the demos. I think we can just pass all data into the template and have a single page that renders all variations. 

I will try and knock up a quick branch to explain what I mean